inc/common.class.php

```diff
@@ -49,6 +55,11 @@ public static function convertToGiB($size) {
       return "$size $unit";
    }
 
+   /**
+    * Display massive actions
+    * @param Object $massiveactionparams
```

Comment:
I guess the param is an array. 
If it is actually an object, then prefer the class name of this object; this may be useful to add type hinting later